config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_runjobs.run.cnrm.cloud.google.com.yaml

```diff
@@ -969,45 +1011,49 @@ spec:
 
-                  When a new Job is created, or an existing one is updated, Cloud Run will asynchronously perform all necessary steps to bring the Job to the desired state. This process is called reconciliation. While reconciliation is in process, observedGeneration and latest_succeeded_execution, will have transient values that might mismatch the intended state: Once reconciliation is over (and this field is false), there are two possible outcomes: reconciliation succeeded and the state matches the Job, or there was an error, and reconciliation failed. This state can be found in terminalCondition.state.
+                  When a new Job is created, or an existing one is updated, Cloud Run
+                  will asynchronously perform all necessary steps to bring the Job to the
+                  desired state. This process is called reconciliation.
+                  While reconciliation is in process, `observed_generation` and
+                  `latest_succeeded_execution`, will have transient values that might
+                  mismatch the intended state: Once reconciliation is over (and this field is
+                  false), there are two possible outcomes: reconciliation succeeded and the
+                  state matches the Job, or there was an error,  and reconciliation failed.
+                  This state can be found in `terminal_condition.state`.
 
-                  If reconciliation succeeded, the following fields will match: observedGeneration and generation, latest_succeeded_execution and latestCreatedExecution.
+                  If reconciliation succeeded, the following fields will match:
+                  `observed_generation` and `generation`, `latest_succeeded_execution` and
+                  `latest_created_execution`.
 
-                  If reconciliation failed, observedGeneration and latest_succeeded_execution will have the state of the last succeeded execution or empty for newly created Job. Additional information on the failure can be found in terminalCondition and conditions.
+                  If reconciliation failed, `observed_generation` and
+                  `latest_succeeded_execution` will have the state of the last succeeded
+                  execution or empty for newly created Job. Additional information on the
+                  failure can be found in `terminal_condition` and `conditions`.
                 type: boolean
               terminalCondition:
-                description: The Condition of this Job, containing its readiness status,
-                  and detailed error information in case it did not reach the desired
-                  state.
+                description: Output only. The Condition of this Job, containing its
+                  readiness status, and detailed error information in case it did
+                  not reach the desired state.
                 items:
                   properties:
-                    executionReason:
```

Comment:
What happened here?  These look like breaking changes - thankfully only in status, but still

/hold for status breaking changes